berserk/clients/teams.py

```diff
@@ -121,3 +121,13 @@ def search(self, text: str, page: int = 1) -> PaginatedTeams:
         path = "/api/team/search"
         params: Params = {"text": text, "page": page}
         return cast(PaginatedTeams, self._r.get(path, params=params))
+
+    def message_all(self, team_id: str, message: str):
+        """Send a private message to all members of a team. You must own the team.
```

Comment:
I don't believe this is true, one just has to be a leader with the right permissions.

Reply:
Is it wrong on the docs then? ![Screenshot 2023-10-22 at 9 29 06 PM](https://github.com/lichess-org/berserk/assets/24191989/dbcf4d46-a28d-4bc1-a3a3-361f67c686dd)

Reply:
There was a recent change, so now Team Leaders have a more flexible permission system. Presumably you need the "Messages" team leder permission, which allows "Send private messages to all members". Might be similar outdated info in the api docs. (Granted "owner" was also incorrect, it was something any leader could do before)

<img width="863" alt="image" src="https://github.com/lichess-org/berserk/assets/19399877/e73da8f9-1dcc-4354-876c-9137b0ae1214">

Reply:
Updated! Please review 🙏